.github/workflows/build_conda.yml

```diff
@@ -7,12 +7,15 @@ jobs:
   build:
     runs-on: ubuntu-latest
     container:
-      image: continuumio/miniconda3:latest
+      image: ghcr.io/noaa-gfdl/fre-cli:miniconda24.7.1_gcc14.2.0
```

Comment:
Clarifying question for myself, is this a version/image we'll have to update every once in a while?

Reply:
I don't think anything will break anytime soon if we used this image for a while, but it'd be good to get a new image with newer software every year or so, that's what I've typically done with other CI images. Anytime you rebuild the image it'll pull in the latest versions, so you wouldn't have to make any changes to the dockerfile.

Reply:
out of curiosity and my own notekeeping, where can one find the dockerfile for the uploaded container image in question?

Reply:
It's in the hpc-me repo:

https://gitlab.gfdl.noaa.gov/fre/HPC-ME/-/blob/main/ci/fre-cli-gnu/Dockerfile?ref_type=heads